app/interactors/userInteractor.js

```diff
@@ -50,3 +51,15 @@ exports.createUserAdmin = async body => {
 
   return createUser({ ...body, role: 'admin' });
 };
+
+exports.auth0 = async code => {
+  try {
+    const data = await getAuth0Token(code);
+    logger.info('Obtaining Auth0 token...');
+
+    return data;
+  } catch (err) {
+    logger.info('ERROR', err.message);
+    throw auth0Error(err.message);
```

Comment:
Will this overwrite the Auth0 error code? Like 401 or 403

Reply:
Or this is just for user creation?

Reply:
Nope, only if is a problem at axios level, so the Auth0 error has priority. 
This is for token generation. The Auth0 manual that is in the Trello card says that we need to sign up / sign in with the Auth0 service, that gives us a code that we use in that endpoint to obtain the token. So the creation is not at code level, is manual with the URL pre-configured (a bit strange but the doc mention that this is because we don't have a React) Like https://gc-training.us.auth0.com/authorize?client_id=dUW5I3Gz2pwsGlMlMCwryMgr8WdglGoN&response_type=code&connection=google-oauth2&redirect_uri=http%3A%2F%2Flocalhost%3A8081%2Flogin&scope=openid%20profile&state=xyzABC123